R/release.R

```diff
@@ -412,7 +412,7 @@ cran_version <- function(package = project_name(), available = NULL) {
   if (is.null(available)) {
     # Guard against CRAN mirror being unset
     available <- tryCatch(
-      utils::available.packages(repos = default_cran_mirror()),
+      available.packages(repos = default_cran_mirror()),
```

Comment:
Just mildly curious. Does this make a difference?

Updated: oh I see you want to mock it.